templates/icinga2/services.conf.j2

```diff
@@ -303,10 +303,30 @@ apply Service "ups load" {
   vars.datgroup = "ups"
 }
 
+apply Service "systemd" for (systemd => config in host.vars.systemd)  {
+  import "normal-service"
+  check_command = "systemd"
+  vars.notify = false
+
+  vars += config
+  vars.datgroup = "systemd"
+  vars.datitem = systemd
+}
+
 apply ScheduledDowntime "downtime " for (downtime => config in host.vars.downtime) to Service {
   author = "icingaadmin"
   comment = "Configured via ansible"
 
   ranges = config.time
   assign where service.vars.datgroup in config.service_groups
 }
+
+apply Service "ldap" for (ldap => config in host.vars.ldap) {
+  import "normal-service"
+  check_command = "ldap"
+  assign where host.zone == "master"
```

Comment:
Do not add an `assign` statement to an apply definition without very strong reason to do so (e.g. cluster zone sync check or something like that). This will make the check not execute on a host it has been _manually_ configured on if that host doesn't also happen to be a master, which doesn't seem like a good idea.